emukit/bayesian_optimization/loops/bayesian_optimization_loop.py

```diff
@@ -28,6 +29,7 @@ def __init__(self, space: ParameterSpace, model: IModel, acquisition: Acquisitio
                             size is greater than one, this acquisition must output positive values only.
         :param update_interval: Number of iterations between optimization of model hyper-parameters. Defaults to 1.
         :param batch_size: How many points to evaluate in one iteration of the optimization loop. Defaults to 1.
+        :param acquisition_optimizer: Uses gradient based optimizer if None. Defaults to None.
```

Comment:
This should really just say "Optimizer blah blah. Defaults to gradient base optimizer." I am not sure if this has any implications for the code, or just a docstring change.

Reply:
I extended all the docstrings.